### DISCUSSION TITLE
Scheduling Sanity Check

### DISCUSSION THREAD

**Discussion by jdhitsolutions** [buttondown/discussions#35] [Ideas]
As part of the submission checklist, I'd love something that would make sure I'm not double-booking. In other words, if I already have an email scheduled for the same day, warn me if I'm trying to schedule an email for the same day. I have to think few people send out multiple emails on the same day. I have a set schedule and time that I always use. This happened to me today. I scheduled two posts to go out at the same date and time. Its not the end of the world, but a sanity check would be nice.

**Comment by jmduke** [comment]
Love this idea. Moving it to the roadmap.
